e2e/config.go

```diff
@@ -37,8 +40,19 @@ var (
 	lvmOperatorUninstall bool
 	scheme               = runtime.NewScheme()
 	crClient             crclient.Client
+	parameterCodec       runtime.ParameterCodec
+	clientset            *kubernetes.Clientset
 )
 
+type Clientset struct {
+	*corev1.CoreV1Client
+}
+
+// GetParameterCodec is the function used to retrieve the parameterCodec
+func GetParameterCodec() runtime.ParameterCodec {
+	return parameterCodec
```

Comment:
where has the parameterCodec been initialised?

Reply:
Corrected.